screen.css

```diff
@@ -19,10 +19,17 @@
 *
 */
 
-body {
+  body {
   font-family: sans-serif, arial;
   padding: 0;
   margin: 0;
+    -webkit-touch-callout: none;
```

Comment:
Great stuff! But indentation... Please fix! (Sorry, but I'd rather hit you with this nitpicking now than later)

Reply:
@mbitsnbites i put two additional commits and hopefully fixed the indentation :)